react-native-scripts/src/scripts/eject.js

```diff
@@ -98,25 +101,30 @@ Ejecting is permanent! Please be careful with your selection.
         newDisplayName = expName;
       }
 
+      const validateEnteredDisplayname = s => { return s.length > 0; }
+      const validateEnteredName = s => { return s.length > 0 && s.indexOf('-') === -1 && s.indexOf(' ') === -1; }
```

Comment:
I find this difficult to follow. Maybe define them at the end of the file with full function declarations?